### PR TITLE
Disable word editing when practices exist

### PR DESCRIPTION
**Pull request by djrenren**

### CODE REVIEW

File: client/src/components/Lesson/LessonCreator.tsx [hunk 1]
```diff
@@ -7,7 +7,8 @@ import {
     Practice, SectionType
 } from '../../reducers/practiceReducer';
 import {
-    TextField
+    TextField,
+    Toolbar, ToolbarTitle
```
Comment (7imbrook) [inline client/src/components/Lesson/LessonCreator.tsx]: I guess I'm ok with this.

File: client/src/components/Lesson/LessonCreator.tsx [hunk 2]
```diff
@@ -77,6 +78,9 @@ export class LessonCreator extends BindingComponent<LessonCreatorProps> {
 
     render() {
         let content: any;
+        const hasPractices = this.props.practices ? this.props.practices.length > 0 : false;
+        const disabledMessage = hasPractices ?
+            <div style={{padding: '10px', color: 'white', textAlign: 'center', background: '#ff4081'}}> You cannot edit words while practices exist</div> : undefined;
```
Comment (7imbrook) [inline client/src/components/Lesson/LessonCreator.tsx]: Prefer style const at top of file.

File: client/src/components/Lesson/LessonCreator.tsx [hunk 3]
```diff
@@ -91,9 +95,14 @@ export class LessonCreator extends BindingComponent<LessonCreatorProps> {
                             value={this.state['title']}
                             onChange={this.bindValueToName.bind(this)}/>
                     </Page>
-                    <Page title='Word Editor'>
+                    <Page header={<div>
+                     <Toolbar><ToolbarTitle text='Word Editor'/></Toolbar>
+                     {disabledMessage}
```
Comment (7imbrook) [inline client/src/components/Lesson/LessonCreator.tsx]: Tabbing

File: client/src/components/Lesson/WordCreator/WordDetails.tsx [hunk 4]
```diff
@@ -64,19 +68,22 @@ export class WordDetails extends BindingComponent<WordDetailsProps> {
...
+                    hintText='Antonyms'
+                    disabled={this.props.disabled}/>
 
             <h3>Forms</h3>
             <WordFormSelector
                 forms={this.props.wordInfo.forms || []}
                 onChange={this.onFormChange.bind(this)}
                 onNewValueChange={ (val: string) => this.setState({ wordFormNewValue: val }) }
                 newValue={this.state['wordFormNewValue'] || ''}
+                disabled={this.props.disabled}
```
Comment (7imbrook) [inline client/src/components/Lesson/WordCreator/WordDetails.tsx:86]: does `...props.disabled` work?

**Review by 7imbrook — APPROVED** [review]
I guess

File: client/src/components/Lesson/WordCreator/WordDetails.tsx (hunk 4 above)
Comment (djrenren) [reply client/src/components/Lesson/WordCreator/WordDetails.tsx:86]: nah :/

**Pull request merged**